frontend/src/containers/TheHeader.vue

```diff
@@ -11,7 +11,7 @@
       @click="$store.commit('toggleSidebarDesktop')"
     />
     <CHeaderBrand class="mx-auto d-lg-none" to="/">
-      <img src="../assets/polaris.svg" class="img-fluid" alt="Responsive image">
+      <img src="../assets/autotracks_blue_crp.png" class="img-fluid" alt="Responsive image">
```

Comment:
Have you tried putting a fixed height style for the image? Or maybe a max height. Here is a potential fix:
`<img src="../assets/autotracks_blue_crp.png" class="img-fluid header-img" alt="Responsive image">`

and in the style tags
```
<styles scoped>
.header-img {
     max-height: 100px;
}
</styles>
```

Reply:
Requested change has now been committed